.travis.yml

```diff
@@ -3,19 +3,33 @@ python:
   - "2.7"
   - "3.3"
 install:
-  - pip install argparse
-  - pip install nose flake8
+  - pip install nose
+  - pip install pep8 pyflakes
 # For now install ament_package from git
   - git clone https://github.com/ament/ament_package.git
   - cd ament_package
   - python setup.py install
   - cd ..
-# Also install a special branch of osrf_pycommon
+# For now install osrf_pycommon from git
   - git clone https://github.com/osrf/osrf_pycommon.git
   - cd osrf_pycommon
   - python setup.py install
   - cd ..
+# For now install ament_copyright / ament_pep8 / ament_pyflakes from git
+  - git clone https://github.com/ament/ament_lint.git -b split_linter_packages_in_python_and_cmake
+  - cd ament_lint/ament_copyright
+  - python setup.py install
+  - cd ../..
+  - cd ament_lint/ament_pep8
+  - python setup.py install
+  - cd ../..
+  - cd ament_lint/ament_pyflakes
+  - python setup.py install
+  - cd ../..
 script:
   - python setup.py nosetests -s
+  - ament_copyright ament_tools scripts test
+  - ament_pep8 ament_tools scripts test
+  - ament_pyflakes ament_tools scripts test
```

Comment:
I would prefer these to be part of the nose tests, as the flake8 test was before. Why did you do it this way instead?